01_prepare_host.sh

```diff
@@ -111,40 +111,40 @@ fi
 
 # Allow local non-root-user access to libvirt
 # Restart libvirtd service to get the new group membership loaded
-if ! id "$USER" | grep -q libvirt; then
-  sudo usermod -a -G "libvirt" "$USER"
+if ! id "${USER}" | grep -q libvirt; then
+  sudo usermod -a -G "libvirt" "${USER}"
   sudo systemctl restart libvirtd
 fi
 
-if [ "${EPHEMERAL_CLUSTER}" == "minikube" ]; then
-  if ! command -v minikube 2>/dev/null || [[ "$(minikube version --short)" != "${MINIKUBE_VERSION}" ]]; then
-      wget --no-verbose -O minikube https://storage.googleapis.com/minikube/releases/"${MINIKUBE_VERSION}"/minikube-linux-amd64
+if [ "${EPHEMERAL_CLUSTER}" = "minikube" ]; then
+  if ! command -v minikube 2>/dev/null || [ "$(minikube version --short)" != "${MINIKUBE_VERSION}" ]; then
```

Comment:
`command -v ...` doesn't print anything on error, but it prints on success. `2>/dev/null` is thus no-op. Use `& >/dev/null` instead.